lib/templates/android-kotlin/inc/_enum_getter_setter.liquid

```diff
@@ -0,0 +1,15 @@
+{%- for attribute in entity.attributes %}
+ 
+    {%- if attribute.realm_read_only.size == 0 and attribute.enum_type.size > 0 %}
+        {%- assign name = attribute.name %}
+        
+    fun  get{{ name | titleize }}Enum(): {{ attribute.enum_type }}? {
```

Comment:
just stylistic sugar, but I think in Kotlin when a getter's implementation is only one line, you can use `fun getFooEnum(): Type = Foo.get(foo)` instead of using curly braces?

Reply:
Indeed